test/basic/cv.jl

```diff
@@ -0,0 +1,51 @@
+module TestFit
```

Comment:
Careful here, you're effectively overwriting a modeule (it probably generates a warning during test runs)

Reply:
o yes thanks for spotting that! will also modify that for the `TestSearchCv` as well in my next push